target/linux/ramips/dts/FON2601.dts

```diff
@@ -0,0 +1,167 @@
+// SPDX-License-Identifier: GPL-2.0
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "fon,fon2601", "ralink,mt7620a-soc";
+	model = "Fon FON2601";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power_r {
+			label = "fon2601:red:power";
+			gpios = <&gpio0 9 GPIO_ACTIVE_LOW>;
+		};
+
+		internet_g {
+			label = "fon2601:green:internet";
+			gpios = <&gpio0 10 GPIO_ACTIVE_LOW>;
+		};
+
+		net_g {
+			label = "fon2601:green:net";
+			gpios = <&gpio0 12 GPIO_ACTIVE_LOW>;
+		};
+
+		wifi_g {
+			label = "fon2601:green:wifi";
+			gpios = <&gpio0 14 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
+
+		reset_wps {
```

Comment:
Just a question: Is this is a multi-function key? Or really just reset?

Reply:
changed to single reset function button at cde53a9 .
This button is displayed as WPS on the enclosure.
However, factory resetting feature is prior to WPS and WPS feature cannot be configured easily by Luci. Therefore, I think that the button is used to dedicated "reset".

Reply:
> This button is displayed as WPS on the enclosure.

Oh, in that case please make this a WPS button. Otherwise this could confuse (end-)users that
grew accustom to the stock-firmware buttons usage. 

The ```hostapd-utils``` package contains the wps button support script. It should be enough to install the "full" wpad (as the -basic or -mini variants do not have WPS enabled) and said package in order to use the push-button feature. (though, the wifi config has to be edited to enable WPS too).

Alternatively,  if you are hell bent on keeping this as the factory reset. Then you should at least document that the buttons function was changed. (I do understand that a factory reset button is important!).